### PR TITLE
Add request router

### PR DESCRIPTION
**Pull request by WojSad**
Signed-off-by: Wojciech Sadowy <wojciech.sadowy@freedomfi.com>

[SAS][Client] Add SAS request router

## Summary

SAS request router is responsible for forwarding requests it receives from the consumer to the SAS server and then passing its response back to the Radio Controller.

## Test Plan

Unit tests attached

### CODE REVIEW

File: configuration_controller/mappings/request_mapping.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  http_method: post
+heartbeatRequest:
+  sas_endpoint: heartbeat
+  http_method: post
+relinquishmentRequest:
+  sas_endpoint: relinquishment
+  http_method: post
+deregistrationRequest:
+  sas_endpoint: deregistration
+  http_method: post
```
Comment (Dysproz) [inline configuration_controller/mappings/request_mapping.yml:19]: Marcin has chosen convention of naming YAML files *.yaml* in deployment tools. Maybe let's stick to one convention across the whole project.

File: configuration_controller/tests/unit/fixtures/fake_mappings/request_mapping.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (Dysproz) [inline configuration_controller/tests/unit/fixtures/fake_mappings/request_mapping.yml:1]: Same as above with YAML naming

**Review by Dysproz — CHANGES_REQUESTED** [review]

File: configuration_controller/tests/unit/fixtures/fake_mappings/request_mapping.yml (hunk 2 above)
Comment (m-trojanowski) [reply configuration_controller/tests/unit/fixtures/fake_mappings/request_mapping.yml:1]: I'll create PR to change files from yaml to yml.

**Review by m-trojanowski — APPROVED** [review]

File: configuration_controller/mappings/request_mapping.yml (hunk 1 above)
Comment (WojSad) [reply configuration_controller/mappings/request_mapping.yml]: Marcin will create a PR changin his extensions to yml

**Pull request merged**